frontend/src/stimulus/controllers/pattern-input.controller.ts

```diff
@@ -226,16 +235,24 @@ export default class PatternInputController extends Controller {
 
   setRealCaretPositionAtNode(target:Node, position:'before'|'after' = 'after'):void {
     const selection = document.getSelection();
-    if (selection) {
-      const postRange = document.createRange();
-      if (position === 'after') {
-        postRange.setStartAfter(target);
-      } else {
-        postRange.setStartBefore(target);
-      }
-      selection?.removeAllRanges();
-      selection?.addRange(postRange);
+    if (selection === null) { return; }
```

Comment:
This is semantically different than the original check, but it looks correct according to documentation. I think this is okay :+1: 

https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection

Reply:
yeah, typescript is way more specific about static typing. So we safely can use that `null` check.